### ISSUE TITLE
Tabjolt - Error loading jvm.dll

### ISSUE THREAD

**Issue by sdasgupta-18** [tableau/TabMigrate#43]
We have performed the below steps to install TabJolt at a client site. We are stuck and need help to take it forward. We have tried our best to solve the error but no luck.
 

- The machine where TabJolt is installed is Windows 2016 with 4 Cores & 10 GB RAM. Java is also installed on the machine.
- Installation of Tabjolt was successful. The inbuilt PostGreSQL database (bundled with the installation) was installed. 
- During the installation, we did not get the configuration dialog box where the tableau server details have to be entered.
- We tried to open the “config.bat” file manually but we were not successful. Then we opened the command prompt and ran the “config.bat” command, but got the below error:
![image](https://user-images.githubusercontent.com/39558562/59197627-6023b000-8baf-11e9-9f6a-532520399675.png)

- As per the link bellow, the error is due to a missing file called MSVC100.dll that should be present under the c:\windows\system32 folder and can be solved by installing the MS VC++ redistributable packages
- https://community.tableau.com/docs/DOC-22488
![image](https://user-images.githubusercontent.com/39558562/59197643-69ad1800-8baf-11e9-9448-156b373c2213.png)

- Typically the MS VC ++ files are already bundled with the installation, but for some reasons it was not successful.
- We downloaded the latest version of MS VC++ redistributable packages (both 32bit and 64bit) and installed it on the server. However the MSVC100.dll file was still not installed/copied under the windows folder.
- Hence we manually downloaded the MSVC100.dll file and copied under the windows folder but still it did not solve the error at step 4 above. The machine was restarted but still getting the same error.
- We ignored this error, and simply tried executing the “Go” command…but we got the same error at the step 4 above.

**Comment by mverette** [comment]
@sdasgupta-18 Did you resolve this issue?  I am experiencing the same behavior in my effort to run a test in TabJolt.  Thanks for any help you can provide given your previous experience with this.

**Comment by mad3lin3** [comment]
@sdasgupta-18 Did you resolve this issue?  I am experiencing the same behavior in my effort to run a test in TabJolt.  Thanks for any help you can provide given your previous experience with this.

**Comment by rahul149386** [comment]
Did you resolve this issue? I am also facing same behavior.

**Comment by sdasgupta-18** [comment]
No, we could not resolve this error. I remember our team had to get a new machine.

**Comment by PWalsh9000** [comment]
I am having a similar issue.  Has anyone resolved it?   The tabjolt is running on my desktop, but attempts to run it on an azure image fail with a simple message that 'c:\TabJolt>jre\bin\java.exe -cp "lib\ext\*" com.tableausoftware.test.tools.TabjoltUI.TabjoltUI
Error: loading: c:\TabJolt\jre\bin\server\jvm.dll'.   There is nothing in the event viewer that sheds light on this problem.

**Comment by gijo404** [comment]
To fix this, update go.bat/config.bat to use java.exe of an external jdk(java 8 worked for me), by default it tries to pick java which reside inside Tabjolt folder and fail.
Pasting content of config.bat (working one)
"D:\jdk\8x64-latest\bin\java.exe" -cp "lib\ext\*" com.tableausoftware.test.tools.TabjoltUI.TabjoltUI
